src/Microsoft.CSharp/pkg/Microsoft.CSharp.pkgproj

```diff
@@ -1,10 +1,7 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
-    <ProjectReference Include="..\ref\Microsoft.CSharp.csproj">
```

Comment:
The description talks about removing depproj's from the pkgproj's. However this is removing the live ref project build, I know we were debating on this one. Did you decide to remove it as opposed to filter it out?